src/data/grammar/grammar.json

```diff
@@ -1,8 +1,5 @@
 {
     "origin": [
-        "#headline#"
```

Comment:
this happened when I resolved the merge, I guess I didn't understand the change as it appeared in the conflict block. I didn't revert it intentionally